fixtures/struct-default-values/README.md

```diff
@@ -0,0 +1,15 @@
+# Default arguments in struct constructors
+
+We support default values for any field in a struct/dictionary.
+Fields with default values can come before fields without any.
+This always worked for Kotlin and Swift.
+Kotlin allows optionally calling with named arguments.
+Swift enforces (by default) named arguments.
+
+However this broke for Python,
+because one cannot specify any non-defaulted arguments after defaulted ones.
+This is now fixed and Python enforces named arguments.
```

Comment:
IIUC this is also a breaking change for Python - is it also a breaking change for Ruby? I'm not sure if that needs to be called out here but certainly does in the CHANGELOG.

Reply:
Oh right, yes. I wrote that readme when I did it first, before I even touched Ruby.
And yes, this needs a changelog entry too